tests/bsim/bluetooth/ll/multiple_id/prj.conf

```diff
@@ -12,8 +12,8 @@ CONFIG_BT_AUTO_DATA_LEN_UPDATE=y
 
 # Maximum simultaneous ACL connections, current maximum considering other roles
 # using ticker nodes count up to 255 (UINT8_MAX) in the hci_ipc with ISO support
-CONFIG_BT_MAX_CONN=233
-CONFIG_BT_ID_MAX=233
+CONFIG_BT_MAX_CONN=231
+CONFIG_BT_ID_MAX=231
```

Comment:
Why is the conn count reduced?

Reply:
As `CONFIG_BT_CTLR_SCAN_AUX_SET` is increased by 2 (that exposes the bug in one of the cap unicast tests), the connection count is reduced by 2. Ticker nodes are limited to `UINT8_MAX`, and when other state/roles use ticker nodes, connection count has to be reduced accordingly.

Reply:
Aha. Could you add that to the commit message? It's not clear from the changes, especially since `CONFIG_BT_CTLR_SCAN_AUX_SET` is being modified in a completely different application (hci_ipc).

Reply:
updated commit message.